bitcash/exceptions.py

```diff
@@ -8,3 +8,7 @@ class InvalidAddress(Exception):
 
 class InvalidNetwork(Exception):
     pass
+
+
+class InvalidEndpointURLProvided(Exception):
+    pass
```

Comment:
This exception should show which endpoint failed.

Reply:
Fixed in 88e3adbd5bd0aae38dd71db3ca0e9fffaea8a340